infra/terraform/modules/kubernetes_access_control/data.tf

```diff
@@ -0,0 +1,129 @@
+data "aws_iam_policy_document" "kubernetes_assume_role_policy" {
+  statement {
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+    principals {
+      type        = "Service"
+      identifiers = ["ec2.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "kubernetes_nodes_aws_iam_role_policy" {
+  statement {
+    effect    = "Allow"
+    actions   = ["ec2:Describe*"]
+    resources = ["*"]
+  }
+
+  statement {
```

Comment:
Do we need these? I remember adding something similar to make the lambda functions log to CloudWatch. Sorry for the silly question, maybe we actually need them.

Reply:
This just allows our instances to ship logs to CloudWatch, which is a useful thing to have. Overall though this IAM policy is the standard policy generated by `kops`, with the single addition of the `AssumeRole` statement to allow `kube2iam` to work. It may be that this policy is more open than is strictly necessary, but again I'm trying to limit the number of changes we make in one go for the time being.